packages/tests/server/websockets.test.ts

```
@@ -1921,3 +1924,214 @@ test('url callback and connection params is invoked for every reconnect', async
+    });
+    await new Promise<void>((resolve) => {
+      const sub = removeMe.connectionState.subscribe({
+        next(state) {
+          if (state.state === 'pending') {
+            resolve();
+            sub.unsubscribe();
+          }
+        },
+      });
+    });
+    console.log('new websocket client created');
```

ESLint

[error] Unexpected console statement.

(no-console)